test/Table_Tests/src/Database/Common_Spec.enso

```diff
@@ -213,10 +213,10 @@ spec prefix connection pending=Nothing =
             t7 = upload "T7" <|
                 Materialized_Table.new [['price', [0.4, 3.5, Nothing, 6.7, Nothing, 97, Nothing]]]
             price = t7.at 'price'
-            price.sum.should_equal 107.6
-            price.min.should_equal 0.4
-            price.max.should_equal 97
-            price.mean.should_equal 26.9
+            price.sum.should_equal 107.6 epsilon=0.000001
```

Comment:
I'd consider extracting this epsilon to a constant, to avoid copying it all over the place.